src/scss/global/base.scss

```diff
@@ -1,3 +1,7 @@
+HTML: {
```

Comment:
Good idea! Remove `:` please

Reply:
Woops, fixed in [a97e27d](https://github.com/yoksel/grid-cheatsheet/pull/3/commits/a97e27ddd4b22036b08690f095bd2e436668c59f) 😁